CoreGraphNode/src/au/gov/asd/tac/constellation/graph/node/gui/SimpleGraphOpener.java

```diff
@@ -87,7 +89,7 @@ protected Object doInBackground() throws Exception {
                     time = System.currentTimeMillis() - t0;
                 } catch (final GraphParseException ex) {
                     gex = ex;
-                    ex.printStackTrace(System.out);
+                    LOGGER.log(Level.WARNING, "{0}", ex.getStackTrace());
```

Comment:
Possibly should be `SEVERE`. Also can you change to the following:
` LOGGER.log(Level.SEVERE, ex.getLocalisedMessage(), ex);`